### ISSUE TITLE
[BUG] DSiWare cia dumping legit selection confusion

### ISSUE THREAD

**Issue by kyoforkshomebrews** [d0k3/GodMode9#851]
I tried dumping all my DSiWare as .cias and I wanted to get both a standard and legit/legit version of each.

When dumping from NAND, I don't get the subselection for pirate or legit/legit and files dumped have the .legit.cia ending.

When dumping from SD I do get the subselection for pirate or legit/legit but upon selecting legit/legit files with ending .piratelegit.cia get dumped.

**Info about your system**
Include this info to make our work easier:
* Console type N3DS
* Anything special about your console? nope, just a standard luma install
* Bootloader boot9strap
* Did you chainload GodMode9 via Luma? Yes

**Help yourself**
*Especially for any kind of boot issue ("GodMode9 doesn't boot")*, but also in many other cases these steps make a lot of sense and we will ask you to do them anyways:
* Check your SD card (using h2testw, f.e.) or try a different one (you wouldn't believe how common failing/fake SD cards are, and what kinds of bugs are caused by them).
* Switch to fastboot3DS using [https://github.com/d0k3/OpenFirmInstaller](OpenFirmInstaller).

**Have you actually read this?**
[X] I have read the information above

**Comment by kyoforkshomebrews** [comment]
TL;DR It seems impossible to get a legit legit from SYSNAND SD?

**Comment by d0k3** [comment]
@kyoforkshomebrews - I guess I will need to do some clarification.

The choice between personalized and generic ticket is a "default" selection. It is intended to stay as it is once selected, and it will only reset after a restart of GodMode9. That's intended, because for most users it is either one or the other, and once selected you'd rarely wish to switch. It's not the best solution for your use case, granted, but I guess as you know now you can handle it.

Next thing: It is, in fact not always possible to get a legit / legit CIA (f.e. from SysNAND SD). That's because we cannot properly locate required ticket data from there.

What I need to take a guess for is steps 4. to 6. in your list. It's evry much possible you selected a title without a personalized ticket (i.e. not installed from the eShop) there.

Does that explain everything? Feel free to follow up with more questions.
